packages/builder/builder-rspack-provider/src/types/config/security.ts

```diff
@@ -1,5 +1,16 @@
-// TODO
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface SecurityConfig {}
+// Todo support security.sri configuration
+/**
+ * Currently, rspack does not support the security.sri configuration.
```

Comment:
@9aoy Can we support security.sri in Rspack?

Reply:
SRI is difficult to support，because `webpack-subresource-integrity` has a hard code dependency on `webpack-html-plugin`.

Reply:
But we can support `security.checkSyntax` & `security.checkPolyfill` next version ~

Reply:
get 👍🏻

Reply:
So, if we can't support security.sri in the near future, we can move the type from builder-share to webpack-provider~

Reply:
+1